patches/minecraft/net/minecraft/world/WorldProvider.java.patch

```diff
@@ -65,12 +65,13 @@
      }
  
      @SideOnly(Side.CLIENT)
-@@ -241,6 +216,336 @@
+@@ -241,6 +216,352 @@
          return new WorldBorder();
      }
  
 +    /*======================================= Forge Start =========================================*/
 +    private net.minecraftforge.client.IRenderHandler skyRenderer = null;
```

Comment:
Could this be renamed to something like `legacySkyRenderer`? It's not public, so binary compatibility wouldn't be lost.

Reply:
I see.